packages/db/postgres/migrations/023-add-user-customer-table.sql

```diff
@@ -0,0 +1,7 @@
+-- add table to keep track of 0..1 customer id for each user
+CREATE TABLE IF NOT EXISTS user_customer
+(
+  id               BIGSERIAL PRIMARY KEY,
+  user_id          BIGINT NOT NULL UNIQUE REFERENCES public.user (id),
```

Comment:
We do queries for user_id, probably we should add an index for it? What you think?

Reply:
good idea! 3df2067c23a0df1465a2555e5dde1f9baaefd8dc